src/components/calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button from './Buttons';
+
+class Calculator extends React.Component {
```

Comment:
* As mentioned by the previous CR it's a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components) to use functional components, kindly convert your component to functional components